src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -410,6 +410,7 @@ private async Task<(long, DatabaseTopology, List<string>)> CreateDatabase(string
                 Server.ServerStore.AssignNodesToDatabase(clusterTopology, databaseRecord);
             }
 
+            databaseRecord.SetSupportedFeatures();
```

Comment:
lets add a test that check that there is no possibility to populate `SupportedFeatures` from Client API, either when we create the database or update existing one

Reply:
changed `SupportedFeatures` to `internal `